incubator/crate/values.yaml

```diff
@@ -0,0 +1,54 @@
+# The Chart for a crateDB deployment with working defaults consists of:
+#   1) A Service (exposing the ports that crate uses)
+#   2) A StatefulSet (initialising the cluster)
+#   3) Resources definition
+app: crate
+
+image:
+  name: crate
+  tag: 3.1.1
+  pullPolicy: IfNotPresent
+
+# 1) The Service connecting 3 ports.
+service:
+  name: crate
+  type: ClusterIP
+
+  adminName: crate-web
+  adminPort: 4200
+  clusterName: cluster
+  clusterPort: 4300
+  # https://crate.io/docs/crate/reference/en/latest/interfaces/postgres.html
+  postgresName: postgres
+  postgresPort: 5432
+
+# 2) The StatefulSet forming the cluster. Read more:
+# https://crate.io/docs/crate/guide/en/latest/scaling/multi-node-setup.html#id10
+# https://crate.io/docs/crate/reference/en/latest/config/environment.html#use-max-50-of-available-ram
+crate:
+  clusterName: my-cluster
+  crateHeapSize: 1g
+  recoverAfterNodes: 1
+  replicas: 1
```

Comment:
Is there any reason you chose 1 replica as the default? I would suggest to make the default 3, since this is the minimum viable setup for a proper CrateDB cluster. In this case, I'd also suggest that the `recoverAfterNodes: 2` and `minimumMasterNodes: 2`.

Reply:
Yes there is. We're also typically using clusters of at least 3 nodes, but we wanted this to work for people experimenting for the first time, typically with a minikube on a single node. A single-node cluster is still a valid cluster and the chart still works. We hope it's clear enough how to scale to 3.